service/matching/version_sets_test.go

```diff
@@ -85,215 +97,422 @@ func mkPromoteInSet(id string) *workflowservice.UpdateWorkerBuildIdCompatibility
 	}
 }
 
+func assertClockGreater(t *testing.T, clock1 clockpb.HybridLogicalClock, clock2 clockpb.HybridLogicalClock) {
+	if clock1.Version == clock2.Version {
```

Comment:
this doesn't work? it fails on [101, 2] > [100, 3] (using [WallClock, Version])

Reply:
Refactored in the next PR.